client/my-sites/stats/stats-purchase/stats-purchase-tier-upgrade-slider.scss

```diff
@@ -1,38 +1,80 @@
+$slider-height: 40px;
+$slider-bg-color: var(--studio-white);
+$thumb-width: 40px;
+$mark-diameter: 8px;
+$mark-border-width: 2px;
+$track-extra-width: ($thumb-width - $mark-diameter - $mark-border-width * 2) / 2;
 
+// The trick is to cover the extra slider track with blocks with the same background color as the slider.
+@mixin slider-track-extra-cover() {
+	content: "";
+	position: absolute;
+	width: $track-extra-width;
+	height: $slider-height;
+	background-color: $slider-bg-color;
+}
 
 .stats-tier-upgrade-slider {
 	--jp-white: #fff;
 	--jp-green-50: #008710;
 	--gray-gray-5: #dcdcde;
+	--studio-gray-40: #787c82;
+	// TODO: Determine correct colors.
+	// Theme or Jetpack? The following are theme colors.
+	--color-accent: #008763;
+	--color-primary: #0675c4;
 	margin-top: 42px;
+	background-color: $slider-bg-color;
+
+	border: 1px solid var(--gray-gray-5);
+	border-radius: 5px; // stylelint-disable-line scales/radii
+	padding: 12px;
 
-	.jp-components-pricing-slider__thumb {
-		border-radius: 4px;
-		border-color: var(--jp-green-50);
-		background-color: var(--jp-green-50);
+	.jp-components-pricing-slider__thumb.upgrade-thumb {
+		width: $thumb-width;
+		height: $slider-height;
+		border-radius: 50%;
+		border-color: var(--color-primary);
+		background-color: var(--color-primary);
 		color: var(--jp-white);
 	}
 
 	.jp-components-pricing-slider__track {
+		// Assuming 40px tall container.
 		height: 4px;
+		top: 18px;
```

Comment:
If this was a variable related to `$slider-height` it would match the approach. e.g. `top: ($slider-height / 2) - $mark-border-width`.

Reply:
I think it should be `$slider-height / 2 - $track-height / 2` since the track is moved inside the container and adjusted position with self-height.

Reply:
I'll get this added today.